docs/src/pages/getting-started/learn/learn.md

```diff
@@ -37,7 +37,7 @@ Here are some recommended resources, some of which are free.
   - 📝 [The blog post](https://medium.com/codingthesmartway-com-blog/getting-started-with-material-ui-for-react-material-design-for-react-364b2688b555)
```

Comment:
Note that most of the learning content is outdated with v5. I'm not even sure keeping v4 content here is a net positive.